dmoj/generator.py

```diff
@@ -42,13 +42,6 @@ def find_runtime(languages):
             compiler_time_limit = compiler_time_limit or clazz.compiler_time_limit
             clazz = type('Executor', (clazz,), {'compiler_time_limit': compiler_time_limit})
 
-        if hasattr(clazz, 'flags'):
-            flags += ['-DWINDOWS_JUDGE', '-DWIN32'] if os.name == 'nt' else ['-DLINUX_JUDGE']
```

Comment:
Shouldn't we still `-DLINUX_JUDGE` for compat?

Reply:
We have precisely 2 problems (`linked` and `ds4`) that rely on it. I don't think it's worth maintaining this hack just for them.

Reply:
`ds4` has it to maintain parity with the judge's compiler so people don't attempt something like
```cpp
#ifdef LINUX_JUDGE
#include<bits/extc++.h>
#endif
```

Reply:
What? Why would the generator matter at all?